Assets/Project/Scripts/UnityChanControlScript.cs

```diff
@@ -4,12 +4,21 @@
 
 namespace UnityChan
 {
+    public class Ability
+    {
+        public string AnimParamName { get; set; }
+        public KeyCode Key { get; set; }
+        public Action FuncToCall { get; set; }
+        public bool IsEnabled { get; set; }
+    }
+
     [RequireComponent(typeof(Animator))]
     [RequireComponent(typeof(CapsuleCollider))]
     [RequireComponent(typeof(Rigidbody))]
     public class UnityChanControlScript : MonoBehaviour
     {
         #region Fields
```

Comment:
Consider replacing these #regions with [Header("xxxx")]

Reply:
I see, if can be both used with region then I'll iclude, I just like the effect of region that auto collapses  when opening the file so its clean to see, hopefully header does the same, will try